UI/styles/style.css

```diff
@@ -0,0 +1,138 @@
+body{
+  margin:0;
+  padding:0;
+}
+
+.container{
+width:80%;
+margin:auto;
+}
+
+header{
+  height: 3rem;
+}
+nav{
+  position: relative;
+}
+
+.navigation__wrapper{
+  position: fixed;
+  padding: 1rem 0;
+  top: 0;
+  left: 0;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  background: white;
+  box-shadow: 0px 5px 5px -4px rgba(0, 0, 255, 0.1);
+  z-index: 3;
+}
+
+#header-title{
+  float: left;
+  font-weight: 900;
+  letter-spacing: 1.5px;
+  font-size: 30px;
+  line-height: 1.3;
+  margin:0px;
+  padding: 0px 0px;
+}
+
+.nav-link{
+  float: right;
+  padding: 5px 15px 0 15px;
+}
+
+.nav-link a:hover{
+  font-weight: 600;
+}
+
+.navigation__wrapper a{
+  text-decoration: none;
+  color: #117d5a;
+}
+
+/* The dropdown container */
+  .dropdown {
+    overflow: hidden;
+  }
+  
+  /* Dropdown button */
+  .dropdown .dropbtn {
+    font-size: 25px; 
+    border: none;
+    outline: none;
+    color: green;
+    background-color: inherit;
+    font-family: inherit; 
+    margin: 0; 
+  }
+  
+  /* Add a red background color to navbar links on hover */
+  .navbar a:hover, .dropdown:hover .dropbtn {
+   /* background-color: green;*/
```

Comment:
It looks good to me but you still have some commented code, remove the commented and unused code then you are free to merge the PR.

Reply:
alright, thanks..